tests/lib/wheel.py

```diff
@@ -51,15 +51,13 @@ class Default(Enum):
 Defaulted = Union[Default, T]
 
 
-def ensure_binary(value):
-    # type: (AnyStr) -> bytes
+def ensure_binary(value: AnyStr) -> bytes:
```

Comment:
`AnyStr` can be replaced with just `str` (it’s a Python 2-3 compatibility thing)

Reply:
Updated in #10345.